### ISSUE TITLE
Error when using createNewRelicPlugin

### ISSUE THREAD

**Issue by RSFelix** [newrelic/newrelic-node-apollo-server-plugin#277]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Error when using createNewRelicPlugin
When trying to start createNewRelicPlugin I am getting the following error:

```sh
This expression is not callable.
  Type 'typeof import("/home/application/node_modules/@newrelic/apollo-server-plugin/index")' has no call signatures.ts(2349)
```

This is my index.ts:

```JS
import { ApolloServer, type ApolloServerPlugin } from '@apollo/server'
import { ApolloServerPluginInlineTrace } from '@apollo/server/plugin/inlineTrace'

const newRelicPlugin = createNewRelicPlugin<ApolloServerPlugin>({})

const server = new ApolloServer({
  typeDefs,
  resolvers,
  plugins: [
    newRelicPlugin,
  ]
})
```
As stipulated in the documentation.

## My Configs

tsconfig.json
```json
{
  "compilerOptions": {
    "module": "NodeNext",
    "removeComments": true,
    "lib": [
      "es2020"
    ],
    "target": "es2020",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "strictNullChecks": true,
    "moduleResolution": "NodeNext",
    "allowSyntheticDefaultImports": true
  }
}
```

package.json
```json
{
  "name": "analysts_graphql_api",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "type": "module",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "compile": "tsc",
    "start": "npm run compile && cp ./src/schema.gql ./dist/schema.gql && node -r newrelic ./dist/index.js",
    "start:kafka": "npm run compile && node -r newrelic ./dist/kafka.js"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@apollo/server": "^4.7.1",
    "@newrelic/apollo-server-plugin": "^4.0.0",
    "body-parser": "^1.20.2",
    "cors": "^2.8.5",
    "dataloader": "^2.2.2",
    "express": "^4.18.2",
    "graphql": "^16.6.0",
    "graphql-redis-subscriptions": "^2.6.0",
    "graphql-subscriptions": "^2.0.0",
    "graphql-tools": "^9.0.0",
    "graphql-ws": "^5.14.0",
    "ioredis": "^5.3.2",
    "jsonwebtoken": "^9.0.1",
    "jwks-rsa": "^3.0.1",
    "kafkajs": "^2.2.4",
    "newrelic": "latest",
    "protos_js": "github:BB-KR-project/protos_js#20230905-8b0715a1a6cb279e0f115ca606323cacad5a58be",
    "ws": "^8.13.0"
  },
  "devDependencies": {
    "@types/cors": "^2.8.13",
    "@types/express": "^4.17.17",
    "@types/node": "^20.2.5",
    "@types/ws": "^8.5.5",
    "@typescript-eslint/eslint-plugin": "^5.59.7",
    "eslint": "^8.41.0",
    "eslint-config-standard-with-typescript": "^34.0.1",
    "eslint-plugin-import": "^2.27.5",
    "eslint-plugin-n": "^15.7.0",
    "eslint-plugin-promise": "^6.1.1",
    "typescript": "^5.2.0-dev.20230622"
  }
}

```

I found a previous problem, already closed with the same request, but without a definitive answer or resolution.
Link: https://github.com/newrelic/newrelic-node-apollo-server-plugin/issues/71

I would like help understand what could be happening.
Thank you in advance!

**Comment by bizob2828** [comment]
@RSFelix could you provide a basic reproduction case? It may be fixed with #276 which was just released as 4.0.1 but hard to tell

**Comment by ceolinrenato** [comment]
Hey @bizob2828 I work on the same codebase as @RSFelix I can confirm it was fixed by updating to the 4.0.1 version. Will message him to confirm so we can close this issue

**Comment by bizob2828** [comment]
I'll just close it. If for some reason it does not address your issue, please just reopen with a basic repro.

**Comment by RSFelix** [comment]
Hello @bizob2828, everything is fine?
Thank you for the response. Confirm that it was fixed by updating to version 4.0.1
